style.css

```diff
@@ -483,3 +483,294 @@ hr {
+    background-image: url(media/responsive-background.png);
+    background-repeat: no-repeat;
+    background-position: 100% 50%;
+  }
+
+  .background {
+    display: none;
+  }
+
+  .container .links,
+  .top {
+    display: flex;
+    flex-direction: column;
+    order: -1;
+    justify-content: center;
+  }
+
+  .container.paragraph {
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: center;
+  }
+
+  /* ----Project Container------------ */
+  .project-container {
+    display: grid;
+    padding: 20px;
+  }
+
+  .project-container hr {
+    border: 1px solid  #212121;
+    width: 60%;
+    margin-top: -130px;
+    margin-left: 510px;
+  }
+
+  .project-heading {
+    text-align: justify;
+    padding-left: 150px;
+  }
+
+  .img {
+    float: left;
+    width: 684px;
+    height: 448px;
+    padding: 50px;
+    margin-top: 2px;
+    margin-left: 60px;
+  }
+
+  .info-title,
+  .info-detail,
+  .lang {
+    justify-items: right;
+    margin-top: 50px;
+  }
+
+  .btn {
+    float: left;
+    background: #ff6b00;
+    color: #fff;
+    width: 120px;
+    height: 48px;
+    align-items: center;
+    text-align: center;
+    padding: 12px;
+    margin: 30px;
+    border: none;
+    font-family: 'Inter', sans-serif;
+    font-size: 17px;
+    font-weight: bold;
+    line-height: 24px;
+    cursor: pointer;
+    transition-duration: 00.4s;
+  }
+
+  .project-display {
+    display: flex;
+    flex-wrap: wrap;
+    justify-content: space-evenly;
+  }
+
+  .firstproject {
+    background-image: url(media/project.png);
+    opacity: 1;
+  }
+
+  .secondproject,
+  .fifthproject {
+    background-image: url(media/layout2nd.png);
+    opacity: 1;
+  }
+
+  .thirdproject,
+  .sixthproject {
+    background-image: url(media/layout3rd.png);
+    opacity: 1;
+  }
+
+  .forthproject {
+    background-image: url(media/Layout.png);
+    opacity: 1;
+  }
+
+  .contact-form form ul {
+    width: 90%;
+  }
+
+  .firstproject:first-child ul,
+  .firstproject:first-child h3,
+  .firstproject:first-child p {
+    visibility: hidden;
+  }
+
+  .display .btn-dis {
+    display: none;
+  }
+
+  .display:first-child .btn-dis {
+    display: block;
+  }
+
+  /* -------------About Me---------------------- */
+
+  .about-me {
+    background-image: url(media/illustration-about\ me@2x.png);
+    background-position: 100% 15%;
+    background-size: 35%;
+    border: hidden;
+  }
+
+  .about-me-heading,
+  .about-me-content,
+  .btn-about-me {
+    display: flex;
+    text-align: justify;
+    margin-left: 150px;
+    margin-right: 150px;
+  }
+
+  .about-me-heading {
+    font-size: 72px;
+    line-height: 81px;
+  }
+
+  .about-me-content {
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  /* ---------------Programming Language------------ */
+
+  .prg {
+    display: grid;
+    grid-template-columns: repeat(3, 1fr);
+    align-items: baseline;
+    margin-bottom: 120px;
+    width: 1024px;
+    margin-left: 200px;
+  }
```

Comment:
- [ ] The Icons beside the titles are far away, please make the margin of each be closer to the attributed title.

![,,n,n](https://user-images.githubusercontent.com/61562626/134549896-dbf66c84-de6e-4b7c-9360-e362c5ecb2f1.jpeg)

Reply:
Could you help me to understand how to fix that as  in live demo link the icons are close to the title.

Reply:
You can add margin.

Reply:
I didn't use icons separately, I used list-style-image for each icon